get-started.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Getting Started
+summary: Quick list of instructions and hints to get started with NServiceBus and the Particular Service Platform
+reviewed: 2016-12-05
+---
+
+There is a variety of starting points for NServiceBus and the platform. The path taken will depend on an individual's specific experience and preferences for learning.
+
+
+## High Level Content
+
+These articles are recommended to gain an understanding of the key concepts and technologies used within the Platform.
+
+ * [Service Platform Overview](/platform/): Provides an overview of the various parts of Platform; [NServiceBus](/nservicebus/), [ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/) and [ServicePulse](/servicepulse/).
+ * [Architectural Principles](/nservicebus/architecture/principles.md)
+ * [Concepts Overview](/nservicebus/concept-overview.md): A high level overview of the concepts, features and vernacular of NServiceBus.
+ * [Hosting](/nservicebus/hosting/): Provides information on the various approaches to hosting an instance of NServiceBus.
+
+
+## Tutorials
+
+The [Messaging Basics Tutorial](/tutorials/nservicebus-101/) gives a guided walk though of building an NServiceBus system and introduces the basic concepts of messaging. The tutorial also has a downloadable and runnable solution to illustrate the finished product.
+
+
+## Go straight to runnable code
+
+Samples take a "download first" approach and then provided an explanation of how the given scenario, technology or concept.
+
+Go to [Introduction to Sample](/samples/) or [Skip to the list of samples](/samples/#related-samples).
+
+There are some specific samples that will help get started:
+
+ * [Step by Step Sample](/samples/step-by-step/): Illustrates essential NServiceBus concepts by showing how to build a simple system.
+ * [Endpoint configuration choices](/samples/endpoint-configuration/): Walks through the most common choices required when creating a first endpoint. It will also show the configuration APIs needed to implement those choices.
+
+
+## Getting the bits
+
+The library parts of NServiceBus (.NET assemblies) are deployed through [NuGet Packages](https://www.nuget.org). The majority of the NuGet packages are listed under the [NServiceBus NuGet User](https://www.nuget.org/profiles/nservicebus) with the main library is deployed via the [NServiceBus NuGet package](https://www.nuget.org/packages/NServiceBus/).
```

Comment:
I'd explain first that NServiceBus consists of a number of elements: transport (abstraction over a low-level messaging technology like MSMQ or SQL Server), persistance (used by features like publish-subscribe, sagas, etc.), optional others (callbacks, etc.). It might be not that obvious at first.